GameDev/PlayMenu.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
+#include "MenuBase.h"
+#include "LTexture.h"
+#include "LevelConfig.h"
+class MenuState;
+class PlayMenu : public MenuBase
+{
+private:
+	map<string, bool>* levelsMap;
+	LevelConfig levelconfig;
+	int hoverX;
+	int hoverY;
+	int selectedCounter;
+	SDL_Renderer* renderer;
+	MenuState* mainMenu;
+
+	LTexture background;
+	//background
+	SDL_Rect backgroundRect;
+	//background
+	SDL_Texture* backgroundTexture;
+
+	SDL_Rect pos[6];
+
+	SDL_Color textColor;
+	SDL_Color hoverTextColor;
+	SDL_Color diseabledTextColor;
+
+	TTF_Font* titleFont;
+	TTF_Font* textFont;
+
```

Comment:
kunnen al die rectangles en textures niet misschien in een map of enige collectie? Er zijn er namelijk hartstikke veel en dat maakt het niet misschien overzichtelijker, ook ivm. aanmaken, cleanup / destructen enzo, gewoon loopen door de collectie